connector_importer_demo/models/res_partner.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.addons.component.core import Component
+
+
+class PartnerRecordHandler(Component):
```

Comment:
this shall not be needed. 
Modify importer._init_importer as this:
```py
class RecordImporter(Component):
[...]
    _record_handler_usage = 'odoorecord.handler'
    _tracking_handler_usage = 'tracking.handler'

    def _init_importer(self, recordset):
        [...]
        self.record_handler = self.component(usage=self._record_handler_usage)
        [...]
        self.tracker = self.component(usage=self._tracking_handler_usage)
```
Then on the CSV importer declare the new usage.

Then you can trash this file completely and for all the imports the std stuff will be used ;)
Then if you'll define some specific importer/mapper they applied on a specific model they will be used anyway.